handler/dimensions.go

```diff
@@ -30,6 +31,81 @@ func NewDimensions(cfg *config.Config, r responder, c cantabularClient, d datase
 	}
 }
 
+// GetDimensionCategories is the handler for GET/population-types/{population-type}/dimension-categories?dims=xxx
+func (h *Dimensions) GetDimensionCategories(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	req := contract.GetDimensionCategoriesRequest{
+		PopulationType: chi.URLParam(r, "population-type"),
+		Variables:      r.URL.Query().Get("dims"),
```

Comment:
The ParseRequest function also parses URL params tagged with the struct tag 'schema'. e.g.

```
type FooRequest{
  Variables string `schema: "dims"`
}

Reply:
Yes I actually did add the schema tag, this is a hangover, I can remove the line